### PR TITLE
[plugin.program.dnsleaktest] 1.1

### PR DESCRIPTION
**Pull request by Space2Walker**
### Description
This addon performs a DNS leak Test on bash.ws and displays the Result.
I hope this is OK.
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.program.dnsleaktest/addon.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.program.dnsleaktest"
+version="1.1"
```
Comment (basrieter) [inline plugin.program.dnsleaktest/addon.xml]: I guess this indentation could be nicer here.

File: plugin.program.dnsleaktest/addon.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+</extension>
+<extension point="xbmc.addon.metadata">
+  <summary lang="en">DNS Leak-Test Plugin for Kodi</summary>
+  <description lang="en_GB">This addon performs a DNS leak test on https://bash.ws/dnsleak!
+  The Basic Code comes from https://github.com/macvk/dnsleaktest.
+  </description>
+  <disclaimer lang="en_GB">Use add your own risk!!</disclaimer>
+  <platform>all</platform>
+  <license>GPL-3.0</license>
+  <website>https://bash.ws</website>
```
Comment (basrieter) [inline plugin.program.dnsleaktest/addon.xml]: This should be your website, not the site from bash.ws, unless it is actually Bash.ws that is creating this add-on. You could add your github page here.

File: plugin.program.dnsleaktest/addon.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.program.dnsleaktest"
+version="1.1"
+name="DNS Leak-Test"
+provider-name="Space2Walker">
+<requires>
+  <import addon="xbmc.python" version="2.26.0"/>
+  <import addon="script.module.certifi" version="2017.07.27.1"/>
```
Comment (basrieter) [inline plugin.program.dnsleaktest/addon.xml]: You only need the `requests` import, not others. These are dependencies of the `requests` module.

File: plugin.program.dnsleaktest/main.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                              + " [" + dns_server['country_name']
+                              + "]")
+
+            else:
+                set_label(str(dns_server['ip']))
+
+
+if __name__ == '__main__':
+
+    xbmcplugin.setPluginCategory(_handle, 'APP')
```
Comment (basrieter) [inline plugin.program.dnsleaktest/main.py]: Why this specific Category?

File: plugin.program.dnsleaktest/addon.xml (hunk 2 above)
Comment (Rechi) [reply plugin.program.dnsleaktest/addon.xml]: Website should point to the site which provides the service, so this is correct.

Comment (basrieter) [reply plugin.program.dnsleaktest/addon.xml]: Looking at other add-ons and their website, you seem to be right.

**Comment by enen92** [comment]
ping @Space2Walker

**Comment by Space2Walker** [comment]
sorry for the late response, very little time at the moment.
the category is for some skins, but i don't remember which one.
the addon.xml is fixed
a readme.md is a good thing but im not a native speaker, so its a big task for me, maybe later

**Pull request merged**